### PR TITLE
[AIRFLOW-2845] Remove asserts from the contrib package

### PR DESCRIPTION
**Pull request by xnuinside**
Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [AIRFLOW-2845](https://issues.apache.org/jira/projects/AIRFLOW/issues/AIRFLOW-2845) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-XXX
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:
`asserts` is used in Airflow contrib package code .  And from point of view for which purposes asserts are really is, it's not correct.

If we look at documentation we could find information what asserts is debug tool: https://docs.python.org/3/reference/simple_stmts.html#the-assert-statement and also it is could be disabled globally by default. 

So, I just want to change debug asserts to ValueError and TypeError.
### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:
It's covered by existing tests. No new features or important changes. 

### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [x] In case of new functionality, my PR adds documentation that describes how to use it.
  - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.

### Code Quality

- [x] Passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

**Comment by xnuinside** [comment]
it's little PR, just for clean code

File: airflow/contrib/hooks/bigquery_hook.py [hunk 1]
```diff
@@ -1518,14 +1517,17 @@ def _bq_cast(string_field, bq_type):
     elif bq_type == 'FLOAT' or bq_type == 'TIMESTAMP':
         return float(string_field)
     elif bq_type == 'BOOLEAN':
-        assert string_field in set(['true', 'false'])
+        if string_field not in (['true', 'false']):
```
Comment (Fokko) [inline airflow/contrib/hooks/bigquery_hook.py]: This can be done without the braces `'vo' not in ['true', 'false']`

**Review by Fokko — CHANGES_REQUESTED** [review]
Thanks @xnuinside for making the code look nicer.

One minor nit, please take a look.

@kaxil do you have any comments on the Google stuff?

**Comment by kaxil** [comment]
@Fokko That looks fine. I wasn't entirely sure about `assert` just being a debug tool. But looks like it is from the link provided.

**Comment by ashb** [comment]
Most of the time `assert` is enabled in Python, but it _can_ be disabled so shouldn't be relied on for critical checks.

File: airflow/contrib/hooks/bigquery_hook.py (hunk 1 above)
Comment (xnuinside) [reply airflow/contrib/hooks/bigquery_hook.py]: done :)

**Review by Fokko — APPROVED** [review]

**Comment by Fokko** [comment]
Thanks @xnuinside

**Comment by xnuinside** [comment]
thanks all for the review :)

**Pull request merged**